### DISCUSSION TITLE
Custom MPD Form

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#1815] [Fixed Issues]
@rern 
I tried restricting the output format using the User's Configuration feature, but the entry does not appear in /etc/mpd.conf.
Is that how it should be?

**Comment by kan-ibal** [comment]
The best way for me is to use custom config file. I put:
`include "/home/custom-mpd.conf"`
in the rAudio Users Configurations and keep all my settings in custom-mpd.conf file.

**Reply by R101G** [reply]
Thanks @kan-ibal, that seems a good solution, although I am not sure how it would deal with entries which conflict with the rAudio entries. I want a bit perfect output for 16 bit sources, but any others to be properly dithered down rather than truncated. I have no idea what the mp3 or aac decoder outputs are.

**Reply by kan-ibal** [reply]
Have you seen [Bit-perfect playback](https://mpd.readthedocs.io/en/stable/user.html#bit-perfect-playback)?

**Reply by R101G** [reply]
Yes, that is what it is doing at the moment, and is what I want for 16 bit sources. If the data is more than 16 bit, I would like to have it dithered down properly, rather than truncated. As most of my sources are 16 bit, it's not important, but is just something I would like to play with. The first thing is to check if a format setting for 16 bits still gives bit-perfect output with 16 bit sources, but it looks as if adding a format entry might get overruled by the rAudio defaults.
